tmp/pr356_test_documentation_validation.py

```
@@ -0,0 +1,651 @@
+            workflow_path = Path(f'.github/workflows/{workflow_file}')
+            assert workflow_path.exists(), \
+                f"Documented workflow not found: {workflow_file}"
+    
+    def test_workflow_changes_documented(self):
+        """Verify major workflow changes are documented."""
+        # Get workflow files
+        workflow_dir = Path('.github/workflows')
+        if not workflow_dir.exists():
+            pytest.skip("Workflow directory not found")
+        
+        workflows = list(workflow_dir.glob('*.yml'))
```

Ruff (0.14.6)

Local variable `workflows` is assigned to but never used

Remove assignment to unused variable `workflows`

(F841)